resources/lib/KodiHelper.py

```diff
@@ -192,6 +193,31 @@ def show_no_seasons_notification (self):
         dialog = xbmcgui.Dialog()
         dialog.notification(self.get_local_string(string_id=30010), self.get_local_string(string_id=30012))
         return True
+        
+    def show_really_remove (self, title, type, year):
+        """Ask user for yes / no
+
+        Returns
+        -------
+        bool
+            Answer yes/no
+        """
+        dialog = xbmcgui.Dialog()
+        if year == '0000':
+            return dialog.yesno(self.get_local_string(string_id=30047),title)
+        return dialog.yesno(self.get_local_string(string_id=30047),title+' ('+str(year)+')')
+
+    def show_local_fb_updated (self):
```

Comment:
Shouldn't that be `show_local_db_updated` instead of `show_local_fb_updated` or am I missing something?